RecoMuon/TrackerSeedGenerator/plugins/TSGForOIFromL2.cc

```diff
@@ -0,0 +1,474 @@
+/**
+  \class    TSGForOIFromL2
+  \brief    Create L3MuonTrajectorySeeds from L2 Muons updated at vertex in an outside-in manner
+  \author   Benjamin Radburn-Smith, Santiago Folgueras, Bibhuprasad Mahakud, Jan Frederik Schulte (Purdue University, West Lafayette, USA)
+ */
+
+#include "RecoMuon/TrackerSeedGenerator/plugins/TSGForOIFromL2.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "DataFormats/Math/interface/deltaR.h"
+
+#include <memory>
+
+using namespace edm;
```

Comment:
Are these using declarations ever needed? I have the impression that you define ```std::``` and ```edm::``` wherever is needed